build.sbt

```diff
@@ -1,10 +1,6 @@
 import com.typesafe.tools.mima.plugin.{MimaPlugin, MimaKeys}
 
-resolvers in ThisBuild += "scala-pr" at "https://scala-ci.typesafe.com/artifactory/scala-pr-validation-snapshots/"
```

Comment:
Is the new version supposed be in some other repo, too? Because it is in this one.